plugin/src/software/aws/toolkits/eclipse/amazonq/views/model/QDeveloperProfile.java

```diff
@@ -30,8 +31,9 @@ public final IdentityDetails getIdentityDetails() {
         return this.identityDetails;
     }
 
+    @JsonIgnore
     public final String getRegion() {
-        return identityDetails.region();
+        return identityDetails != null ? identityDetails.region() : null;
```

Comment:
nit: iirc, this field is mostly `null`, should be more reliable to get region from the arn
https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/profile/QRegionProfile.kt#L12-L16